src/stats/driver.rs

```diff
@@ -54,8 +54,10 @@ impl Default for AccumulatedStatsInner {
 impl AccumulatedStatsInner {
     pub async fn update(&mut self, message: Arc<Protocol>) {
         self.last_update = chrono::Utc::now().timestamp_micros() as u64;
-        self.bytes += message.raw_bytes().len();
-        self.messages += 1;
-        self.delay += self.last_update - message.timestamp;
+        let _ = self.bytes.wrapping_add(message.raw_bytes().len() as u64);
+        let _ = self.messages.wrapping_add(1);
+        let _ = self
+            .delay
+            .wrapping_add(self.last_update - message.timestamp);
```

Comment:
lol, I got confused with the function notation

```suggestion
        self.bytes = self.bytes.wrapping_add(message.raw_bytes().len() as u64);
        self.messages = self.messages.wrapping_add(1);
        self.delay = self
            .delay
            .wrapping_add(self.last_update - message.timestamp);
```